server/etcdserver/v3_server.go

```diff
@@ -314,6 +323,9 @@ func (s *EtcdServer) LeaseRenew(ctx context.Context, id lease.LeaseID) (int64, e
 }
 
 func (s *EtcdServer) LeaseTimeToLive(ctx context.Context, r *pb.LeaseTimeToLiveRequest) (*pb.LeaseTimeToLiveResponse, error) {
+	if r.ID < int64(lease.NoLease) {
```

Comment:
```suggestion
	if r.ID <= int64(lease.NoLease) {
```

You should not be able to get TTL for NoLease